specification.md

```diff
@@ -3770,7 +3838,7 @@ You can also use the filter property to filter the state data after data is inje
         ]
      },
      "stateDataFilter": {
-        "output": "${ {people: [.people[] | select(.age < 40)]} }"
+        "output": "${people: [.people[] | select(.age < 40)]}"
```

Comment:
invalid expression

Reply:
yes, thx, the search/replace struck back again.